lib/base/model.js

```diff
@@ -642,17 +658,20 @@ ModelBase.prototype.timestamp = function(options) {
   const createdAtKey = timestampKeys[0];
   const updatedAtKey = timestampKeys[1];
   const isNewModel = method === 'insert';
-  const setUpdatedAt = updatedAtKey && this.hasChanged(updatedAtKey)
+  const setUpdatedAt = updatedAtKey && this.hasChanged(updatedAtKey);
 
-  if (updatedAtKey && (isNewModel && !setUpdatedAt || this.hasChanged() && !setUpdatedAt)) {
+  if (
+    updatedAtKey &&
+    ((isNewModel && !setUpdatedAt) || (this.hasChanged() && !setUpdatedAt))
```

Comment:
I also dislike all these unnecessary `()`, although I have to admit it makes the code more readable. Is this configurable? If not we can go with it.

Reply:
it is not, sadly. I also dislike this :(

Reply:
Ok, not a big deal then.